go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/edgexfoundry/go-mod-bootstrap v0.0.60
 	github.com/edgexfoundry/go-mod-configuration v0.0.8
-	github.com/edgexfoundry/go-mod-core-contracts v0.1.119
+	github.com/edgexfoundry/go-mod-core-contracts v0.1.121-0.20201204060101-93bcc6a22012
```

Comment:
Please note that github.com/edgexfoundry/go-mod-core-contracts module had bumped major version to [v2.0.0-dev.1](https://github.com/edgexfoundry/go-mod-core-contracts/tree/v2.0.0-dev.1).  Per [go module documentation](https://golang.org/cmd/go/#hdr-Module_compatibility_and_semantic_versioning) quoted below, we should update the go.mod of go-mod-core-contracts to include v2 as part of module, e.g. **github.com/edgexfoundry/go-mod-core-contracts/v2**.  

> modules with major version v2 or later use a module path with that major version as the final element. For example, version v2.0.0 of example.com/m must instead use module path example.com/m/v2, and packages in that module would use that path as their import path prefix, as in example.com/m/v2/sub/pkg.

Otherwise we will encounter an error like 
```
require github.com/edgexfoundry/go-mod-core-contracts: version "v2.0.0-dev.1" invalid: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
```
I intentionally used pseudo version here to work around the v2 module issue here.  We shall update go.mod later once we have solution for the v2 module issue.

Reply:
Note that the version change has been reversed, for now.

Reply:
https://github.com/edgexfoundry/edgex-go/pull/2897/commits/38007af085b1abde05f68ea8e42d8924bf981647
updated the go.mod to use github.com/edgexfoundry/go-mod-core-contracts v0.1.121